turnstile-example/turnstile/examples/cs478/2021-02-18/stlc.rkt

```diff
@@ -1,10 +1,14 @@
 #lang turnstile/quicklang
 
-(provide λ Int → + #%datum ascribe)
+(provide λ Int Bool Unit unit → + #%datum ascribe #%app if succ pred iszero begin2)
 
-(define-base-type Int)
+(define-base-types Int Bool Unit)
 (define-type-constructor → #:arity = 2)
 
+(define empty-list '())
```

Comment:
I wanted to create a `unit` type *and* value like in the book, so I hacked it with `define-typed-variable-rename` and adding a case for it in the `#%datum` type rule. This probably doesn't work the same way it does in the book though is my guess. How would you achieve this?